aexpect/ops_linux.py

```diff
@@ -432,6 +432,29 @@ def grep(session, expr, path, check=False, flags="-a"):
     return output
 
 
+def grep_pipe(session, cmd, pattern, flags = "", check = False):
+    """
+    Invoke ``grep`` remotely searching for an expression in a command output.
+
+    :param session: vm session
+    :type session: ShellSession
+    :param str cmd: command that its output will be grepped
+    :param str pattern: pattern to grep
+    :param str flags: extra flags passed to ``grep`` on the command line
+    :param bool check: whether to quietly run grep for a boolean check
```

Comment:
Again, it'd be nice to unify this with the existing `grep` by changing the position of `check` and `flags`.

Something like `grep_pipe(session, cmd, expr, check, flags)` would be IMO better (or even `session, expr, cmd, check, flags` but logically the `cmd` should be first here. Not a strong preference over those two, though.)